src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -18,7 +18,7 @@
   private static final String PAUSE_CLASS_NAME = "pause-overlay";
   private static final String REPLAY_CLASS_NAME = "replay-overlay";
   private static final String CURRENT_TIME_CLASS_NAME = "current-time";
-  private static final String SPEAKER_CLASS_NAME = "speaker";
+  private static final String SPEAKER_CLASS_NAME = "volume-button,porvata-switchable-icon";
```

Comment:
I think it should looks like `volume-button.porvata-switchable-icon` buuutt... much cleaner will be change when you define SPEAKER_CLASS_NAME as `.volume-button.porvata-switchable-icon` with `.` at the beginng

Reply:
ups good catch itshould be "."